### DISCUSSION TITLE
```yield``` and Collection Expression Syntax

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#8845] [Language Ideas] [answered]
There have been many requests for a ```yield all list;```.
With collection expression syntax, an alternative syntax might be:
```yield return ..list;```

**Comment by HaloFour** [comment, marked as answer]
I don't think the syntax has ever been the problem as much as the behavior, the quadratic nature of nested iterators.

**Reply by colejohnson66** [reply]
For reference, see https://github.com/dotnet/roslyn/issues/15 and #378
